lua/hover/providers/gh_user.lua

```diff
@@ -5,7 +5,7 @@ local async = require('hover.async')
 ---@return string
 local function get_user()
   local WORD = fn.expand('<cWORD>')
-  local user = WORD:match('TODO%(@?(.*)%):')
+  local user = WORD:match '%f[%w_]@?(.*)%f[^%w_]'
```

Comment:
What is this expected to match in practice? Some comments would help.

Reply:
I don't think the regex is perfect, but it matches `@user` and `user`, the `%f[%w_]` and `%f[^%w_]` is because of the lack of `\b` in Lua. https://stackoverflow.com/a/32854326/213124

Reply:
This will cause a lot of false-positives as most words exist as a gihutb user.

Reply:
@hello @world @some @user

Reply:
The plugin depends on an explicit action (moving the cursor on a word and then triggering the hover) so the false-positives would be minimal. But we can make the `@` mandatory too, so it matches only `@user`

Reply:
Making `@` mandatory instead of optional might be a good middle ground then?